src/index.ts

```diff
@@ -0,0 +1,105 @@
+/// <reference path="./bcoin.d.ts" />
+
+/** Orchestrate the hello swap apps
+ * Start 2 instances of helloSwap
+ * "Link" them together. eg extract PeerId etc.
+ * Call "sendSwap()" on one instance which is going to trigger a swap query
+ * Assumptions:
+ * - Use internal bitcoind wallet, simple utxo manipulation
+ * - Use Parity dev wallet to fund 2 wallets, could be replaced with Ethereum wallet: https://github.com/coblox/bobtimus/issues/78
+ */
+
+import { formatEther, parseEther } from "ethers/utils";
+import { BitcoinWallet } from "./bitcoinWallet";
+import { EthereumWallet } from "./ethereumWallet";
+import { HelloSwap } from "./helloSwap";
+import LedgerActionHandler from "./ledgerActions";
+import { setupBitcoin, setupEthereum } from "./setup/setup";
+
+const BITCOIND_P2P_URI = "127.0.0.1:18444";
+
+// TODO: Funding should be handled by the create-comit-app
+async function startApp(
+    whoAmI: string,
+    cndUrl: string,
+    startBitcoin: number,
+    startEther: string
+) {
+    const bitcoinWallet = new BitcoinWallet("regtest");
+    await bitcoinWallet.init(BITCOIND_P2P_URI);
+    await new Promise(r => setTimeout(r, 1000));
+    if (startBitcoin) {
+        await setupBitcoin(await bitcoinWallet.getAddress(), startBitcoin);
+        await new Promise(r => setTimeout(r, 10000));
+    }
+    const ethereumWallet = new EthereumWallet();
+    if (startEther) {
+        await setupEthereum(
+            ethereumWallet.getAccount(),
+            parseEther(startEther)
+        );
+    }
+
+    const ledgerActionHandler = new LedgerActionHandler(
+        bitcoinWallet,
+        ethereumWallet
+    );
+
+    const app = new HelloSwap(
+        cndUrl,
+        ethereumWallet.getAccount(),
+        whoAmI,
+        ledgerActionHandler,
+        () => true
+    );
+    console.log(`[${whoAmI}] Started:`, await app.cndPeerId());
```

Comment:
```suggestion
    console.log(`[${whoAmI}] Started cnd, peer id:`, await app.cndPeerId());
```
... or just act on my overall comment on moving tech-info to a `debug` log (or similar).